src/api/operator/operatorApi.ts

```diff
@@ -162,3 +168,57 @@ export async function getFeeQuote(networkId: Network, tokenAddress: string): Pro
     return response.json()
   }
 }
+
+/**
+ * Gets a single order by id
+ */
+export async function getOrder(params: GetOrderParams): Promise<RawOrder> {
+  const { networkId, orderId } = params
+
+  console.log(`[getOrder] Fetching order id '${orderId}' on network ${networkId}`)
+
+  const queryString = `/orders/${orderId}`
+
+  const response = await _get(networkId, queryString)
+
+  if (!response.ok) {
+    throw new Error(`Request failed: [${response.status}] ${await response.text()}`)
+  }
+
+  const order: RawOrder = await response.json()
+
+  return order
+}
+
+/**
+ * Gets a list of orders
+ * Optional filters:
+ *  - owner: address
+ *  - sellToken: address
+ *  - buyToken: address
+ */
+export async function getOrders(params: GetOrdersParams): Promise<RawOrder[]> {
```

Comment:
how are you handling these `async` methods if it throws for any reason not relayed to `!response.ok` - could this ever happen? I would suggest `try/catch` if so

Reply:
try catching the hell out of them now

still throwing errors though, so it's up to the caller to decide what to do with them

I intend to wrap those calls into hooks that'll handle the errors